### ISSUE TITLE
Exception when right click on tab to split editor from elm plugin (but in other filetype)

### ISSUE THREAD

**Issue by wadouk** [klazuka/intellij-elm#347]
```
com.intellij.openapi.extensions.impl.PicoPluginExtensionInitializationException: org.frawa.elmtest.run.ElmTestRunConfigurationProducer has unsatisfied dependency: class org.frawa.elmtest.run.ElmTestRunConfigurationType among unsatisfiable dependencies: [[class org.frawa.elmtest.run.ElmTestConfigurationFactory], [class org.frawa.elmtest.run.ElmTestRunConfigurationType]] where DefaultPicoContainer (root) was the leaf container being asked for dependencies.
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.getComponentInstance(ExtensionComponentAdapter.java:90)
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.getExtension(ExtensionComponentAdapter.java:113)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processAdapters(ExtensionPointImpl.java:252)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.getExtensionList(ExtensionPointImpl.java:179)
	at com.intellij.openapi.extensions.ExtensionPointName.getExtensionList(ExtensionPointName.java:32)
	at com.intellij.execution.actions.RunConfigurationProducer.getProducers(RunConfigurationProducer.java:34)
	at com.intellij.execution.actions.ConfigurationContext.findExisting(ConfigurationContext.java:237)
	at com.intellij.execution.actions.BaseRunConfigurationAction.update(BaseRunConfigurationAction.java:190)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:171)
	at com.intellij.openapi.actionSystem.impl.Utils.doUpdate(Utils.java:202)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:151)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:177)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:177)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:177)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:177)
	at com.intellij.openapi.actionSystem.impl.Utils.fillMenu(Utils.java:295)
	at com.intellij.openapi.actionSystem.impl.ActionPopupMenuImpl$MyMenu.show(ActionPopupMenuImpl.java:109)
	at com.intellij.ui.tabs.impl.TabLabel.handlePopup(TabLabel.java:406)
	at com.intellij.ui.tabs.impl.TabLabel.access$100(TabLabel.java:47)
	at com.intellij.ui.tabs.impl.TabLabel$1.mousePressed(TabLabel.java:99)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:279)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:279)
	at java.awt.Component.processMouseEvent(Component.java:6545)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3325)
	at java.awt.Component.processEvent(Component.java:6313)
	at java.awt.Container.processEvent(Container.java:2237)
	at java.awt.Component.dispatchEventImpl(Component.java:4903)
	at java.awt.Container.dispatchEventImpl(Container.java:2295)
	at java.awt.Component.dispatchEvent(Component.java:4725)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4889)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4523)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4467)
	at java.awt.Container.dispatchEventImpl(Container.java:2281)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4725)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:764)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:90)
	at java.awt.EventQueue$4.run(EventQueue.java:737)
	at java.awt.EventQueue$4.run(EventQueue.java:735)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:734)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:719)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:664)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:363)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: org.picocontainer.defaults.UnsatisfiableDependenciesException: org.frawa.elmtest.run.ElmTestRunConfigurationProducer has unsatisfied dependency: class org.frawa.elmtest.run.ElmTestRunConfigurationType among unsatisfiable dependencies: [[class org.frawa.elmtest.run.ElmTestConfigurationFactory], [class org.frawa.elmtest.run.ElmTestRunConfigurationType]] where DefaultPicoContainer (root) was the leaf container being asked for dependencies.
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.getGreediestSatisfiableConstructor(CachingConstructorInjectionComponentAdapter.java:166)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.doGetComponentInstance(CachingConstructorInjectionComponentAdapter.java:74)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.instantiateGuarded(CachingConstructorInjectionComponentAdapter.java:62)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.getComponentInstance(CachingConstructorInjectionComponentAdapter.java:45)
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.getComponentInstance(ExtensionComponentAdapter.java:72)
	... 55 more
Caused by:

org.picocontainer.defaults.UnsatisfiableDependenciesException: org.frawa.elmtest.run.ElmTestRunConfigurationProducer has unsatisfied dependency: class org.frawa.elmtest.run.ElmTestRunConfigurationType among unsatisfiable dependencies: [[class org.frawa.elmtest.run.ElmTestConfigurationFactory], [class org.frawa.elmtest.run.ElmTestRunConfigurationType]] where DefaultPicoContainer (root) was the leaf container being asked for dependencies.
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.getGreediestSatisfiableConstructor(CachingConstructorInjectionComponentAdapter.java:166)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.doGetComponentInstance(CachingConstructorInjectionComponentAdapter.java:74)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.instantiateGuarded(CachingConstructorInjectionComponentAdapter.java:62)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.getComponentInstance(CachingConstructorInjectionComponentAdapter.java:45)
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.getComponentInstance(ExtensionComponentAdapter.java:72)
	at com.intellij.openapi.extensions.impl.ExtensionComponentAdapter.getExtension(ExtensionComponentAdapter.java:113)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.processAdapters(ExtensionPointImpl.java:252)
	at com.intellij.openapi.extensions.impl.ExtensionPointImpl.getExtensionList(ExtensionPointImpl.java:179)
	at com.intellij.openapi.extensions.ExtensionPointName.getExtensionList(ExtensionPointName.java:32)
	at com.intellij.execution.actions.RunConfigurationProducer.getProducers(RunConfigurationProducer.java:34)
	at com.intellij.execution.actions.ConfigurationContext.findExisting(ConfigurationContext.java:237)
	at com.intellij.execution.actions.BaseRunConfigurationAction.update(BaseRunConfigurationAction.java:190)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:171)
	at com.intellij.openapi.actionSystem.impl.Utils.doUpdate(Utils.java:202)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:151)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:177)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:177)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:177)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:177)
	at com.intellij.openapi.actionSystem.impl.Utils.fillMenu(Utils.java:295)
	at com.intellij.openapi.actionSystem.impl.ActionPopupMenuImpl$MyMenu.show(ActionPopupMenuImpl.java:109)
	at com.intellij.ui.tabs.impl.TabLabel.handlePopup(TabLabel.java:406)
	at com.intellij.ui.tabs.impl.TabLabel.access$100(TabLabel.java:47)
	at com.intellij.ui.tabs.impl.TabLabel$1.mousePressed(TabLabel.java:99)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:279)
	at java.awt.AWTEventMulticaster.mousePressed(AWTEventMulticaster.java:279)
	at java.awt.Component.processMouseEvent(Component.java:6545)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3325)
	at java.awt.Component.processEvent(Component.java:6313)
	at java.awt.Container.processEvent(Container.java:2237)
	at java.awt.Component.dispatchEventImpl(Component.java:4903)
	at java.awt.Container.dispatchEventImpl(Container.java:2295)
	at java.awt.Component.dispatchEvent(Component.java:4725)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4889)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4523)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4467)
	at java.awt.Container.dispatchEventImpl(Container.java:2281)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4725)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:764)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:90)
	at java.awt.EventQueue$4.run(EventQueue.java:737)
	at java.awt.EventQueue$4.run(EventQueue.java:735)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:734)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:719)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:664)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:363)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

Environment :
IntelliJ IDEA 2018.3.5 (Ultimate Edition)
Build #IU-183.5912.21, built on February 26, 2019
JRE: 1.8.0_152-release-1343-b28 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.19.24-1-MANJARO

**Comment by klazuka** [comment]
@wadouk sorry for the delay. Is it possible that you had previously installed the [elm-test-runner](
https://github.com/frawa/intellij-elm-test-runner) plugin? That code was merged into this plugin as of version 3.0. To prevent a conflict, you must un-install the old test-runner plugin.

**Comment by wadouk** [comment]
@klazuka thanks, much better
